src/Microsoft.SqlTools.ServiceLayer/Profiler/Contracts/PauseProfilingRequest.cs

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Profiler.Contracts
+{
+    /// <summary>
+    /// Pause Profiling request parameters
+    /// </summary>
+    public class PauseProfilingParams : GeneralRequestDetails
+    {
+        public string OwnerUri { get; set; }
+    }
+
+    public class PauseProfilingResult
+    {
+        public bool Succeeded { get; set; }
+
+        public string ErrorMessage { get; set; }
```

Comment:
Do we need the `Succeeded` and `ErrorMessage` properties? I don't see anywhere where `ErrorMessage` is used, and it looks like the code only sends this with `Succeeded` set to true right now.

If neither of them are needed then this class can just be empty - it's not the cleanest code but we do it for other things like `QueryExecuteResult` already.